src/NServiceBus.ContainerTests/When_using_nested_containers.cs

```diff
@@ -148,14 +195,16 @@ public void UoW_components_in_the_parent_container_should_be_singletons_in_the_c
 
```

Comment:
the name of this test if probably one reason for the confusions. should we rename it `[...]_should_be_singletons_in_the_*same*_child_container`?